### ISSUE TITLE
[Proposal] Add user guide, especially about directives

### ISSUE THREAD

**Issue by okuramasafumi** [ruby/rdoc#1119]
I noticed that there's no list for directives. In fact, there's not a comprehensive user guide in English (the only I found was in Japanese: https://docs.ruby-lang.org/ja/latest/library/rdoc.html).

Adding user guide has two purposes. First, of course, users will refer to the guide so that they can use directives and other features of RDoc more effectively. Second, it will be a single source of truth for RDoc behavior so developers can rely on it.

I suggest the first version of the guide could be a direct translation of the Japanese guide mentioned above since we don't have to talk about what should be in the guide.

Any thoughts?

**Comment by BurdetteLamar** [comment]
@okuramasafumi, https://docs.ruby-lang.org/en/master/RDoc/MarkupReference.html#class-RDoc::MarkupReference-label-Directives did not give you what you wanted?

**Comment by okuramasafumi** [comment]
@BurdetteLamar Ah that's it, thanks!
I wonder how we can find it though. From https://docs.ruby-lang.org/en/master/RDoc.html I couldn't find a link to that page.

**Comment by BurdetteLamar** [comment]
@okuramasafumi, good point.  The page (which, you may know, I wrote a couple of years ago) is linked from https://docs.ruby-lang.org/en/master/RDoc/Markup.html#class-RDoc::Markup-label-RDoc+Markup+Reference.  It should also be linked from other pages likely to be found by search engines.  I'll look into that.
